torch/_meta_registrations.py

```diff
@@ -3109,6 +3109,25 @@ def nan_to_num(self, nan=None, posinf=None, neginf=None):
 import torch._refs.nn.functional
 import torch._refs.special
 
+@register_meta(torch.ops.aten.t_)
+def t_(self):
+    ndims = self.ndim
+    assert ndims <= 2
+    size = self.size()
+
+    dim0 = size[0]
+    dim1 = size[1]
+
+    if dim0 == dim1:
+        return self
```

Comment:
This looks wrong. Where did you get this implementation from?

Reply:
transpose_ in TensorShape.cpp

Reply:
Pretty new to writing meta registrations, lmk what's wrong here, and if there's a better place to check / look.